ansible/roles/satellite-configuration/tasks/01_manifest.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: fetching list of existing subscriptions
+  shell: hammer --output cve subscription list --organization "{{ORG}}"
+  register: subscription_list
+  ignore_errors: yes
+  tags:
+    - satellite_configuration
+    - manifest
+- debug: var=subscription_list
+- name: Creating list fact of existing subscriptions
+  set_fact:
+    list_of_existing_subscriptions: "{{ list_of_existing_subscriptions + [ item.split(':').1.lstrip(' ') ] }}"
+  loop: "{{ subscription_list.stdout_lines }}"
+
+- debug: var=list_of_existing_subscriptions
+
+- name: Copying and uploading Menifest
+  when: SUBSCRIPTION_NAME not in list_of_existing_subscriptions
+  tags:
+    - satellite_configuration
+    - manifest
+  block: 
+    - name: Copy manifest
+      copy:
+        src: ./files/{{ MANIFEST_FILE }} 
+        dest: /tmp
+
+    - name: Uploading manifest 
+      shell:  hammer subscription upload  
+        --file /tmp/{{ MANIFEST_FILE }} 
+        --organization "{{ORG}}"
```

Comment:
I don't think the syntax is correct here.

```yaml
- name: Uploading manifest 
      shell:  hammer subscription upload  
        --file /tmp/{{ MANIFEST_FILE }} 
--organization "{{ORG}}"
```

should be

```yaml
- name: Uploading manifest 
  shell:  >-
    hammer subscription upload  
    --file /tmp/{{ MANIFEST_FILE }} 
    --organization "{{ORG}}"
```

Reply:
This works but I have changed as advised